### PR TITLE
Update Audio Bus page for Godot 3.2

### PR DESCRIPTION
**Pull request by skyace65**
Updates the audio bus page for Godot 3.2. All images have been replaced, and all images have gone through png compression. The EQ effect was separated from the other equalizers since it's a base class. And information was added on the new spectrum analyzer effect.

### CODE REVIEW

File: tutorials/audio/audio_buses.rst [hunk 1]
```diff
@@ -162,19 +162,23 @@ Distortion effects make the sound "dirty". Godot offers several types of
 distortion: *overdrive*, *tan* and *bit crushing*. Distortion can be used
 to simulate sound coming through a low-quality speaker or device.
 
-EQ, EQ6, EQ10, EQ21
-~~~~~~~~~~~~~~~~~~~
+EQ
+~~~
```
Comment (akien-mga) [inline tutorials/audio/audio_buses.rst]: There's one character too many here (rst wants things to be really tidy :)).

File: tutorials/audio/audio_buses.rst [hunk 2]
```diff
@@ -162,19 +162,23 @@ Distortion effects make the sound "dirty". Godot offers several types of
...
 
-Godot provides four equalizers with different numbers of bands. An equalizer on
+EQ is what all other equalizers inherit from and should not be used directly.
```
Comment (akien-mga) [inline tutorials/audio/audio_buses.rst]: `AudioStreamEQ` can be used to create your own custom equalizer, so this should be documented.

**Comment by skyace65** [comment]
I've updated the PR. Though I'm not sure if the `~~` under EQ are working now with just two of them.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**